src/dotnet-dev-certs/Program.cs

```diff
@@ -95,7 +116,34 @@ public static int Main(string[] args)
             }
         }
 
-        private static int CheckHttpsCertificate(CommandOption check, CommandOption trust, IReporter reporter)
+        private static int CleanHttpsCertificates(IReporter reporter)
+        {
+            var manager = new CertificateManager();
+            try
+            {
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    reporter.Output("Cleaning HTTPS development certificates from the machine. A prompt might get" +
+                        "displayed to confirm the removal of some of the certificates.");
+                }
+                if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                {
+                    reporter.Output("Cleaning HTTPS development certificates from the machine. This operation might" +
+                        "require elevated privileges. If that is the case, a prompt for credentials will be displayed.");
+                }
+
+                manager.CleanupHttpsCertificates();
+                reporter.Verbose("HTTPS development certificates successfully removed from the machine.");
+                return Success;
+            }
+            catch
+            {
+                reporter.Error("There was an error trying to clean HTTPS development certificates on this machine.");
```

Comment:
> An error occured when attempting to clean HTTPS development certificates. 

Should we also print the exception message? Might be useful to diagnose \ guide users.

Reply:
Sounds reasonable to me.